ciao-controller/controller_test.go

```diff
@@ -1101,109 +1087,6 @@ func testStartWorkload(t *testing.T, num int, fail bool, reason payloads.StartFa
 	return client, instances
 }
 
-func startTestWorkload(t *testing.T, instanceCh chan []*types.Instance, workloadID string, tenantID string, num int) {
-	w := types.WorkloadRequest{
-		WorkloadID: workloadID,
-		TenantID:   tenantID,
-		Instances:  num,
-	}
-	instances, err := ctl.startWorkload(w)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if len(instances) != num {
-		t.Fatalf("Wrong number of instances, expected %d, got %d", num, len(instances))
-	}
-
-	instanceCh <- instances
-}
-
-// NOTE: the caller is responsible for calling Shutdown() on the *SsntpTestClient
-func testStartWorkloadLaunchCNCI(t *testing.T, num int) (*testutil.SsntpTestClient, []*types.Instance) {
```

Comment:
@kaccardi,@rbradford I'm wondering whether we actually want this code, but that it is not being run as it is named incorrectly, i.e., it starts with a lower case t.  It does sort of look like a unit test.